### PR TITLE
Initial build

### PR DESCRIPTION
**Pull request by acholyn**
Initial build of app, most copied into dev branch. Should work ok on AWS - EBS pulls from development now

### CODE REVIEW

**Review by tcouch — APPROVED** [review]
At the moment running `pip install -r requirements.txt` gives an error if you haven't installed a mysql/maridb client. Probably best to have separate development and production requirements files. Can you configure elastic beanstalk to use a specific requirements file?

The Local Development section in the README isn't really about how to work with the existing code - rather it describes how to create a project from scratch.
Should include a note that LOCAL_ENV needs to be set when running the app locally; e.g. `LOCAL_ENV=1 ./manage.py migrate`
Other problems running locally:
* No secret key
* localhost not in allowed hosts
* need to create superuser account locally
I'd suggest separate local and development settings files to work round both these issues. But I'd really like to just run docker compose -f local.yml :-)

Probably a bit early for me to comment on the models, but you need to write tests. Even if at this point it's just test_creat_film, test_create_actor etc.

File: README.md [hunk 1]
```diff
@@ -136,25 +136,28 @@ On the next page, select the VPC you created, check that public IP is enabled, a
...
+## Security Groups
 
 Whilst the environment is being created, you can check if the security group you defined is available in the VPC console. It will be in `Security Groups` under the `Security` tab in the left hand menu.
 
 If it's available, click on it and scroll to the bottom to edit inbound rules. Allow MySQl connections through the security group that was created for the scaling group attached to your environment, you may also want to allow SSH from this group.
 
 Ensure that the security group you defined on the RDS instance (and then selected in the EBS setup) has been set for MySQL connections.
 
-### CodePipeline
+## CodePipeline
```
[the remark this answers is not in the document]
Comment (acholyn) [reply README.md:160]: it should be there, I'll make it a little more obvious

**Comment by acholyn** [comment]
Good point - I've now split the requirements and will update the readme so local dev uses `requirements-base.txt` (easier to do this than try tell EBS not to use `requirements.txt`

I did some conditional stuff for local on another branch...I might just copy it over. I'll look into having separate settings files for local & aws

File: README.md [hunk 2]
```diff
@@ -1,11 +1,59 @@
 # Museum of Dreams Website Project
 
-This website is hosted at http://museumofdreams.eu-west-2.elasticbeanstalk.com/
+The production website is hosted at http://museumofdreamworlds.eu-west-2.elasticbeanstalk.com/
```
Comment (tcouch) [inline README.md:3]: We need to sort out https for this right? Let's create a separate issue for that

File: museum_of_dreams_project/settings.py [hunk 3]
```diff
@@ -32,6 +38,7 @@
     "35.179.22.251",
     "museumofdreams.eu-west-2.elasticbeanstalk.com",
     "museumofdreamworlds.eu-west-2.elasticbeanstalk.com",
+    "localhost",
```
Comment (tcouch) [inline museum_of_dreams_project/settings.py:41]: I think it's best only to add localhost for local. Also DEBUG needs to be false in production

File: mod_app/tests/test_models.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+class TestActor(TestCase):
+    def setUp(self):
+        Actor.objects.create(name="test actor")
+
+    def test_actor_creation(self):
+        actor = Actor.objects.get(name="test actor")
+
+        self.assertTrue(actor)
+        self.assertEqual(actor.name, "test actor")
```
Comment (tcouch) [inline mod_app/tests/test_models.py:28]: Would be good to also test link between actor and film; e.g.
```
actor.films.add(film)
self.assertQuerySetEqual(film.actors.all(),[actor])
```

**Pull request merged**